### PR TITLE
Improve readability (Vol. 2)

### PR DESCRIPTION
**Pull request by Baggz**
This is second iteration to improve readability of complex data structures. Please see https://attributes-kit-pr-235.herokuapp.com/ and try it out!

## Preview

image

### CODE REVIEW

File: src/Requirement/Requirement.js [hunk 1]
```diff
@@ -15,48 +16,34 @@ class Requirement extends React.Component {
...
 
   render() {
-    const styles = this.renderStyles();
-    const requirement = this.getRequirement();
+    const isRequired = this.requirement[0] === 'required';
 
     return (
-      <div style={styles.root}>
-        {this.renderAsterisk(requirement, styles)}
+      <div style={this.style.base}>
```
Comment (danielgtaylor) [inline src/Requirement/Requirement.js]: Is it possible (or wanted) to return nothing here instead of an empty `div` when an item is not required?

Comment (Baggz) [inline src/Requirement/Requirement.js]: @danielgtaylor 

Yep, definitely! I've fixed that in latter commits.

File: src/ObjectProperty/KeyColumn.js [hunk 2]
```diff
@@ -12,17 +13,77 @@ class KeyColumn extends React.Component {
...
+        this.context.includedProperties !== 'show' && this.context.inheritedProperties !== 'show'
+      ) && (
+        this.context.includedProperties !== 'tag' && this.context.inheritedProperties !== 'tag'
+      )
+    );
+
+    if (isPropertyReferenced) {
+      style.keyColumn.paddingLeft = '20px';
+    }
+    */
```
Comment (danielgtaylor) [inline src/ObjectProperty/KeyColumn.js]: Was this commented section purposefully left in?

File: src/Requirement/Requirement.js [hunk 3]
```diff
@@ -35,14 +35,15 @@ class Requirement extends React.Component {
...
-    );
+    if (isRequired) {
+      return (
+        <div style={this.style.base}>
+          required
+        </div>
+      );
+    }
+
+    return null;
```
Comment (danielgtaylor) [inline src/Requirement/Requirement.js]: Whoops, ignore my previous comment about this :smile:

File: src/ObjectProperty/KeyColumn.js [hunk 4]
```diff
@@ -45,20 +53,6 @@ class KeyColumn extends React.Component {
...
-        this.context.includedProperties !== 'show' && this.context.inheritedProperties !== 'show'
-      ) && (
-        this.context.includedProperties !== 'tag' && this.context.inheritedProperties !== 'tag'
-      )
-    );
-
-    if (isPropertyReferenced) {
-      style.keyColumn.paddingLeft = '20px';
-    }
-    */
```
Comment (danielgtaylor) [inline src/ObjectProperty/KeyColumn.js]: Also ignore the last comment regarding this, I suppose 😛 It's hard to review commit-by-commit!

Comment (Baggz) [inline src/ObjectProperty/KeyColumn.js]: Sorrry! :smile:

**Comment by danielgtaylor** [comment]
LGTM :+1: Looks great and works well. Only minor complaints I had were already taken care of in later commits. Awesome job!

**Pull request merged**